### ISSUE TITLE
docker file improvements

### ISSUE THREAD

**Issue by miguelcleon** [ODM2/CZ-Manager#132]
@lsetiawan from issue #123 
The next round of improvements are to allow user to

1. use their own settings file
2. use uwsgi to run as production
3. separating the database from the application

I was thinking to use the model like docker-erddap, where user attach volume within their host machine to specify where the settings files are, and we have some sort of template where we can let user to download and use... What do you think @ocefpaf ?

**Comment by lsetiawan** [comment]
@miguelcleon Alright, the `lsetiawan/odm2admin` docker image in docker hub has been updated.

**Comment by miguelcleon** [comment]
@lsetiawan Yup, I tried it again from scratch, seems to work great!  @emiliom installation through docker is now working :+1:

**Comment by emiliom** [comment]
:+1: Awesome!

**Comment by lsetiawan** [comment]
Going back to this improvement.

Here's what I have in mind.
> 1. use their own settings file

Like `docker-erddap`, We host the core settings file somewhere users can download in a zip. They can then unzip the settings files, edit them however they like it to be. Take advantage of the `-v` to basically overwrite the settings in the Docker container, they can easily change the settings there. 

*I think we should default the base Database to SQLite maybe??*

> 2. use uwsgi to run as production

Technically, probably not need this one, but I think if the container gets shutdown and turned back on, `runserver` doesn't automatically execute.

> 3. separating the database from the application

This is definitely necessary, since Docker containers should only have one application. So a sole ODM2-Admin django application container would be a good idea. I was thinking using Docker compose to connect each pieces? Need more research on this..
